kolibri/plugins/facility/assets/src/modules/classAssignMembers/actions.js

```diff
@@ -11,8 +11,6 @@ export function enrollLearnersInClass(store, { classId, users }) {
       collection: classId,
       user: userId,
     })),
-  }).catch(err => {
-    store.dispatch('handleApiError', err, { root: true });
```

Comment:
Yeah, so now when this operation fails, you won't see the whole-page `AppError` component, but just the snackbar notification, which is what I recommended in the original issue.